COMPONENT_MATURITY.md

```diff
@@ -0,0 +1,170 @@
+
+### Keyboard navigation
+Component is fully navigable with a keyboard.
+
+#### About
+Keyboard navigation is tested using this [keyboard acceptance criteria (Confluence document)](https://confluence.cms.gov/display/CMSWDS/Keyboard+Navigation+Testing "Keyboard acceptance criteria").
+
+#### When to define as 'Not applicable'?
+When the component has no interactive elements like a button or text input a 'Not applicable' state can be applied.
+
+#### How to test?
+Use this [guidance from WebAIM (external link)](https://webaim.org/techniques/keyboard/ "WebAIM keyboard navigation guidance") to navigate components using a keyboard.
+
+## Code
+
+### Storybook
+Component has stories to cover all defined props.
+
+#### About
+[Storybook (external link)](https://storybook.js.org/ "Storybook JS") is an open-source tool for building components in isolation that is public to all Design System users. 
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+Manually interact with the Storybook stories and make sure that all defined props are in some way tested by our storybook story and that the meaningful ones can be changed. Specifically,
+- Event handlers have visible output through Actions or logging
+- Visual and content props can be changed
+- Alternate stories exist for complex component configurations that are common
+
+### Responsive
+Component designed to work in all responsive breakpoints.
+
+#### About
+[All supported breakpoints (internal link)](https://design.cms.gov/foundation/layout-grid/responsive-design?theme=core "Supported breakpoints") in the CMSDS.
+
+#### When to define as 'Not applicable'?
+Should not be used.
+
+#### How to test?
+View in different responsive sizes with [Storybook controls (internal link)](https://design.cms.gov/storybook/ "CMSDS Storybook") or with browser dev tools.
+
+### Spanish translations
+Includes Spanish translations for default text content.
+
+#### About
+[Read more about Internationalization (internal link)](https://design.cms.gov/getting-started/developers/internationalization/?theme=core "Internationalization") in the CMSDS.
+
+#### When to define as 'Not applicable'?
+When the component has not text to apply a translation to.
+
+#### How to test?
+Changing the language in the [Storybook controls (internal link)](https://design.cms.gov/storybook/ "CMSDS Storybook") should change all English content that isn’t provided through a prop
+
+## Design
+
+### Sketch UI-kit
+Includes all Sketch symbols for defined options.
+
+#### About
+CMS uses [Sketch (external link)](https://www.sketch.com/ "Sketch app") as a tool to create designs.
+
+#### When to define as 'Not applicable'?
+When the component is a utility with no visual element.
+
+#### How to test?
+Verify that the responsive code component matches the responsive symbols in Sketch.
```

Comment:
This how to test section seems to be for the responsive section below? Should this be updated to `Verify with code that all props and options that can be used in code have an equal option in Sketch.`

Reply:
Good catch! Thank you, I'll get this updated.